htlcswitch/htlcnotifier.go

```diff
@@ -360,9 +364,11 @@ func (h *HtlcNotifier) NotifyForwardingFailEvent(key HtlcKey,
 // to as part of a forward or a receive to our node has been settled.
 //
 // Note this is part of the htlcNotifier interface.
-func (h *HtlcNotifier) NotifySettleEvent(key HtlcKey, eventType HtlcEventType) {
+func (h *HtlcNotifier) NotifySettleEvent(key HtlcKey,
+	preimage lntypes.Preimage, eventType HtlcEventType) {
```

Comment:
nit: put an extra line here.

According to [our style guide](https://github.com/lightningnetwork/lnd/blob/master/docs/code_contribution_guidelines.md#code-spacing), 
> If one is forced to wrap lines of function arguments that exceed the 80 character limit, then a new line should be inserted before the first stanza in the comment body.

Reply:
done